### ISSUE TITLE
Move to .qmd?

### ISSUE THREAD

**Issue by jepusto** [jepusto/Designing-Simulations-in-R#6]
Move to a Quarto book instead of bookdown? 

Advantages:
- Nicer formatting/typesetting options.
- Better control of page layout (such as tables and figures wider than text width)
- Hover-over citations and footnotes.
- Sidebar footnotes/citations/figures/comments.
- Better handling of numbered equations.

Downsides:
- Another new system to learn.
- Work to convert cross references and stuff.

**Comment by jepusto** [comment]
Okay I'm game to move too, and I see the value of switching sooner rather than later. My only hesitation comes from needing to send sample chapters to publisher. Should we make the switch after getting the sample chapters taken care of?

**Comment by lmiratrix** [comment]
Yes – let’s get a draft off to the publisher inquiry and then do the move

Luke Miratrix
(Send from phone/iPad; sorry for format, brevity, errors, etc.)

**Comment by jepusto** [comment]
Closed with 0312b252930bbaf8ebf30d12de21e87eeac63bb1
